UE/Application/States/ConnectingState.cpp

```diff
@@ -25,6 +25,12 @@ void ConnectingState::handleTimeout(){
     context.setState<NotConnectedState>();
 }
 
+void ConnectingState::handleCallTimeout(common::PhoneNumber phoneNumber){
```

Comment:
To tutaj jest nie potrzebne. To się raczej nigdy nie wykona, a jak się wykona to niedobrze, bo w connectingState łączymy się z BTS.